src/app/components/search/results/results.component.html

```diff
@@ -0,0 +1,104 @@
+<!-- Results -->
+<div>
+  <hr class="govuk-section-break govuk-section-break--xl govuk-section-break--visible" />
+  <div *ngIf="statusText.toLowerCase() === 'bad request' && statusCode === 100">
```

Comment:
I think this might need to work off the `type` in the error response e.g. `CASE_100` means too many results.

Reply:
I also can't help but thinking it could be clearer than using a combination of criteria?